server/src/main/java/org/opensearch/rest/BaseRestHandler.java

```diff
@@ -95,12 +95,6 @@ public final long getUsageCount() {
      */
     public abstract String getName();
 
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public abstract List<Route> routes();
```

Comment:
There are few test cases that unnecessary overwrite `routes()`. IMO, it will be better to delete those overwrites and rely on the default interface implementation.